### PR TITLE
add POST `/eth/v1/beacon/pool/sync_committees`

### PR DESCRIPTION
**Pull request by rolfyone**
added based on https://hackmd.io/@QYHAVYiHRg65pdI_CEm7Eg/syncommitteeapi#POST-ethv1beaconpoolsync_committees

### CODE REVIEW

**Review by mpetrunic — APPROVED** [review]

File: apis/beacon/pool/sync_committees.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+post:
+  operationId: submitSyncCommitteeSignatures
```
Comment (dapplion) [inline apis/beacon/pool/sync_committees.yaml:2]: Should it be `submitPoolSyncCommitteeSignatures` for consistency with other pool `operationId` items?

Comment (rolfyone) [reply apis/beacon/pool/sync_committees.yaml]: updated

**Comment by dapplion** [comment]
@rolfyone Consider adding back ticks to the route in the PR title for consistency with other PRs :)

When a validator submits a signature and that validator is part of multiple subnets, should the node broadcast the signature on all those subnets?

**Comment by rolfyone** [comment]
sorry, i did it for most, i didn't think people really cared either way, but fixed.

**Comment by rolfyone** [comment]
The beacon node should broadcast the signature on all subnets it is applicable to

**Comment by rolfyone** [comment]
> @rolfyone Consider adding back ticks to the route in the PR title for consistency with other PRs :)
> 
> When a validator submits a signature and that validator is part of multiple subnets, should the node broadcast the signature on all those subnets?

The beacon node should broadcast the signature on all subnets it is applicable to

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**